solarforecastarbiter/metrics/tests/test_probabilistic.py

```diff
@@ -42,6 +42,26 @@ def test_brier_skill_score(fx, fx_prob, ref, ref_prob, obs, value):
     assert prob.brier_skill_score(obs, fx, fx_prob, ref, ref_prob) == value
 
 
+@pytest.mark.parametrize("obs,fx,fx_prob,value", [
+    (4, 5, 50, 0.5),
+    (5, 4, 50, 0.5),
+    (2, 10, 80, 6.4),
+    (100, 80, 50, 10.0),
+    (100, 80, 60, 8.0),
+    (np.array([4, 5]), np.array([5, 4]), np.array([50, 50]), 0.5),
+])
+def test_quantile_score(obs, fx, fx_prob, value):
+    assert prob.quantile_score(obs, fx, fx_prob) == value
+
+
+@pytest.mark.parametrize("obs,fx,fx_prob,ref,ref_prob,value", [
+    (4, 5, 50, 3, 50, 1 - 0.5 / 0.5),
+    (100, 80, 60, 80, 50, 1 - 8 / 10),
+])
+def test_quantile_skill_score(obs, fx, fx_prob, ref, ref_prob, value):
+    assert prob.quantile_skill_score(obs, fx, fx_prob, ref, ref_prob) == value
```

Comment:
Is it worth adding some skill score tests for edge cases?

https://github.com/SolarArbiter/solarforecastarbiter-core/blob/13046c8f73e136d81c35d20436779daeb26f26a8/solarforecastarbiter/metrics/tests/test_deterministic.py#L130-L138

https://github.com/SolarArbiter/solarforecastarbiter-core/pull/395/files#diff-19cf284d1bd4c59fcb5b3fa3dcd65980

Reply:
I think that's a good idea. Plus it gives an excuse to push a commit to cause the RTD task to re-run.